docs/build/building-apps/01-app-go-di.md

```diff
@@ -118,6 +118,108 @@ More information on how work `depinject.Configs` and `depinject.Supply` can be f
 https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-alpha.0/simapp/app_v2.go#L114-L146
 ```
 
+### Modifying the `DefaultGenesis`
```

Comment:
IMHO, this is't the right place, as this isn't specific to app-di.
I would create another page on this under `building-apps` 

Additionally, let's make those new docs v0.52 / main ready